sources/postgres/toddl_test.go

```diff
@@ -155,9 +155,9 @@ func TestToExperimentalSpannerType(t *testing.T) {
 			"b": ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Float64}},
 			"c": ddl.ColumnDef{Name: "c", T: ddl.Type{Name: ddl.Bool}},
 			"d": ddl.ColumnDef{Name: "d", T: ddl.Type{Name: ddl.String, Len: int64(6)}},
-			"e": ddl.ColumnDef{Name: "e", T: ddl.Type{Name: ddl.String, Len: ddl.MaxLength}},
-			"f": ddl.ColumnDef{Name: "f", T: ddl.Type{Name: ddl.String, Len: ddl.MaxLength}},
-			"g": ddl.ColumnDef{Name: "g", T: ddl.Type{Name: ddl.Json}},
+			"e": ddl.ColumnDef{Name: "e", T: ddl.Type{Name: ddl.Numeric}},
+			"f": ddl.ColumnDef{Name: "f", T: ddl.Type{Name: ddl.Date}},
+			"g": ddl.ColumnDef{Name: "g", T: ddl.Type{Name: ddl.String, Len: ddl.MaxLength}},
```

Comment:
JSON is being checked as a String. Is this intentional ?

Reply:
Yes, JSON is not supported in PG Spanner. Thus, we convert JSON to String for PostgreSQL as the target dialect. 

As a side note based on our offline discussion, we don't think we should take up the work for validating the Date in this PR for the cases when we send it as String to Spanner. I have reverted the changes for special handling of Date and created a separate Github issue which tracks Date validation in HarbourBridge. Our handling is not complete anyway today and we only accept ISO 8601 format.